charts/incubator/tt-rss/values.yaml

```
@@ -0,0 +1,179 @@
+          nginx:
+            mountPath: /var/www/html
+            readOnly: true
+        updater:
+          updater:
+            mountPath: /var/www/html
+  config:
+    enabled: true
+    targetSelector:
+        main:
+          main:
+            mountPath: /opt/tt-rss/config.d
+        updater:
+          updater:
+            mountPath: /opt/tt-rss/config.d
+  # plugins:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/plugins.local
+  # themes:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/themes.local
+cnpg:
+  main:
+    enabled: true
+    user: tt-rss
+    database: tt-rss
+portal:
+  open:
+    enabled: true
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+            readiness:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+            startup:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+          env:
+            TTRSS_SELF_URL_PATH: ""
+            # tt-rss tests with "-z" so an empty string should still allow the randomly generated password to be made.
+            ADMIN_USER_PASS: ""
+            TTRSS_DB_NAME: "{{ .Values.cnpg.main.database }}"
+            TTRSS_DB_USER: "{{ .Values.cnpg.main.user }}"
+            TTRSS_DB_PORT: "5432"
+            TTRSS_DB_PASS:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            TTRSS_DB_HOST:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: host
+  nginx:
+    enabled: true
+    type: Deployment
+    podSpec:
+      containers:
+        nginx:
+          enabled: true
+          primary: true
+          imageSelector: nginxImage
+          probes:
+            readiness:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+              # path: "{{ .Values.workload.nginx.podSpec.containers.nginx.env.APP_BASE }}/public.php?op=healthcheck"
+              # port: "{{ .Values.service.main.ports.main.targetPort }}"
+            liveness:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+              # path: "{{ .Values.workload.nginx.podSpec.containers.nginx.env.APP_BASE }}/public.php?op=healthcheck"
+              # port: "{{ .Values.service.main.ports.main.targetPort }}"
+            startup:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+          env:
+            # APP_UPSTREAM: "tt-rss-app.ix-tt-rss.svc.cluster.local"
+            APP_UPSTREAM: '{{ printf "%v-app.ix-%v.svc.cluster.local" (include "tc.v1.common.lib.chart.names.fullname" $) (include "tc.v1.common.lib.chart.names.fullname" $) }}'
```

```suggestion
            APP_UPSTREAM: '{{ printf "%s-app" (include "tc.v1.common.lib.chart.names.fullname" $) }}'
```
This is enough for same namespace resolution.

Also hardcoding `ix-` in namespace is not ok. its not the only platform the charts are used!